oauth2-server/src/main/java/com/clouway/oauth2/UserInfoController.java

```diff
@@ -31,14 +31,14 @@ public Response handleAsOf(Request request, DateTime instantTime) {
     Optional<BearerToken> possibleTokenResponse = tokens.findTokenAvailableAt(accessToken, instantTime);
 
     if (!possibleTokenResponse.isPresent()) {
-      return OAuthError.invalidToken();
+      return OAuthError.invalidToken("Access token not found.");
```

Comment:
"Access token was not found."
to match the identity error.

Reply:
Changed